polliwog/polyline/_try_inflection_points.py

```diff
@@ -53,7 +53,7 @@ def main():
             )
             result_points.append(result)
         except ValueError:
-            pass
+            return None
```

Comment:
Satisfying the tooling is not really a sufficient reason to make a change – it needs to make sense independently. Could you explain it?

Reply:
True... how about printing out a comment before passing, is that reasonable here?

Reply:
@paulmelnikow